.env.production

```diff
@@ -0,0 +1,4 @@
+REACT_APP_CHAIN_ID="1"
+NEXT_PUBLIC_REACT_APP_INFURA_ID=4ec7cc5b5f684e3fba7ba2e0c1533d91
+NEXT_PUBLIC_OPENSEA_API_KEY=83be70605c2441f399b283690a29ee8c
+NEXT_PUBLIC_DEV_MODE=wat
```

Comment:
please remove this and add this file to gitignore. we set these for production via vercel, and we don't want them publicly visible.

Reply:
fasho fasho